embedded-wallets/sdk/android/README.mdx

```diff
@@ -181,7 +182,7 @@ override fun onNewIntent(intent: Intent?) {
 
 After instantiating Web3Auth, the next step is to initialize it using the `initialize` method. This method is essential for setting up the SDK, checking for any active sessions, and fetching the whitelabel configuration from your dashboard.
 
-Once the `initialize` method executes successfully, you can use the `getPrivKey` or `getEd25519PrivKey` methods to verify if an active session exists. If there is no active session, these methods will return an empty string; otherwise, they will return the respective private key.
+Once the `initialize` method executes successfully, you can use the `getPrivateKey` or `getEd25519PrivateKey` methods to verify if an active session exists. If there is no active session, these methods will return an empty string; otherwise, they will return the respective private key.
```

Comment:
This is changed, the `initialize` method will throw an error that needs to be swallowed if the session is not present. 

`initialize` method can fail if the session is not present, or if there is an network issue to get the dashboard configuration, or session validation.

Reply:
added extra context